src/test/suite/fixtures/constants.fixture.ts

```diff
@@ -0,0 +1,63 @@
+interface Fixture {
+  input: string;
+  expectedOutput: string[];
+}
+
+export const subwordFixture: Fixture[] = [
+  {
+    input: "QuickBrownFox",
+    expectedOutput: ["Quick", "Brown", "Fox"],
+  },
+  {
+    input: "quickBrownFox",
+    expectedOutput: ["quick", "Brown", "Fox"],
+  },
+  {
+    input: "quick_brown_fox",
+    expectedOutput: ["quick", "brown", "fox"],
+  },
+  {
+    input: "QUICK_BROWN_FOX",
+    expectedOutput: ["QUICK", "BROWN", "FOX"],
+  },
+  {
+    input: "quick-brown-fox",
+    expectedOutput: ["quick", "brown", "fox"],
+  },
+  {
+    input: "QUICK-BROWN-FOX",
+    expectedOutput: ["QUICK", "BROWN", "FOX"],
+  },
+  {
+    input: "quick.brown.fox",
+    expectedOutput: ["quick", "brown", "fox"],
+  },
+  {
+    input: "QUICK.BROWN.FOX",
+    expectedOutput: ["QUICK", "BROWN", "FOX"],
+  },
+  {
+    input: "../quick/brown/.fox",
+    expectedOutput: ["quick", "brown", "fox"],
+  },
+  {
+    input: "../QUICK/BROWN/.FOX",
+    expectedOutput: ["QUICK", "BROWN", "FOX"],
+  },
+  {
+    input: "quick::brown::fox",
+    expectedOutput: ["quick", "brown", "fox"],
+  },
+  {
+    input: "QUICK::BROWN::FOX",
+    expectedOutput: ["QUICK", "BROWN", "FOX"],
+  },
+  {
+    input: "APIClientFactory",
+    expectedOutput: ["APIClientFactory"],
```

Comment:
Oh wow it chokes really hard on this.  Didn't think it would be that bad 😅.  Worth at least opening an issue and leaving a comment on this and the next one mentioning the issue

Reply:
Yeah and handling it properly will probably be a pain. The current matcher just skips "API" entirely. I've opened #79 and added a comment to these cases.